data/mods/Aftershock/mobs/uplifted_monsters.json

```diff
@@ -3,7 +3,7 @@
     "id": "mon_uplifted_bear",
     "type": "MONSTER",
     "name": { "str": "Schwarz Walder" },
-    "description": "Schwarz Walders are an uplifted animal originally developed by a German company as forest rangers for the expanded Black Forest Conservation Area.  Like all Uplifts they have a human level of intelligence and are indentured laborers to the corporation that grew and raised them, until they reach age of enfranchisement dictated by UICA regulations or they earn their citizenship early.  Shortly afterward the creation of the first Schwarz Walders it was determined that they also made excellent long-range reconnaissance units that could operate independently indefinitely.  This was one of the most popular Uplift models before the Discontinuity, and many frontier planets still retain a significant feral and otherwise independent populations.  They are skilled marksmen and this one appears to be armed with an smg.",
+    "description": "Schwarz Walders are a bipedal uplfited animal originally developed by a German company as forest rangers for the expanded Black Forest Conservation Area.  Like all Uplifts they have a human level of intelligence and are indentured laborers to the corporation that grew and raised them, until they reach age of enfranchisement dictated by UICA regulations or they earn their citizenship early.  Shortly afterward the creation of the first Schwarz Walders it was determined that they also made excellent long range reconnaissance units that could operate independently indefinitely.  This was one of the most popular Uplift models before the Discontinuity, and many frontier planet still retain significant feral and otherwise independent populations.  They are skilled marksmen and this one appears to be armed with an smg.",
```

Comment:
```suggestion
    "description": "Schwarz Walders are a bipedal uplifted animal originally developed by a German company as forest rangers for the expanded Black Forest Conservation Area.  Like all Uplifts they have a human level of intelligence and are indentured laborers to the corporation that grew and raised them, until they reach age of enfranchisement dictated by UICA regulations or they earn their citizenship early.  Shortly afterward the creation of the first Schwarz Walders it was determined that they also made excellent long range reconnaissance units that could operate independently indefinitely.  This was one of the most popular Uplift models before the Discontinuity, and many frontier planet still retain significant feral and otherwise independent populations.  They are skilled marksmen and this one appears to be armed with an smg.",
```